libavpipe/src/avpipe_xc.c

```diff
@@ -2162,7 +2220,8 @@ encode_frame(
         output_packet->pts += params->start_pts;
         output_packet->dts += params->start_pts;
 
-        if ((decoder_context->is_mpegts || decoder_context->is_srt) &&
+        // Detect missing frames for UDP-based live sources
+        if ((is_live_source_udp(decoder_context)) &&
```

Comment:
Should we also detect gaps for general live sources? It seems like it might also be helpful in case there are issues on the source end of the connection-based protocols.

Reply:
Yes but I didn't want to make this kind of change with this PR.  We need to revisit the general area of holes in the live stream.